### ISSUE TITLE
What about country code?

### ISSUE THREAD

**Issue by ArthurStam** [CSSEGISandData/COVID-19#954]
Country code helps identify country. It is usefull for data visualisation, when you want to draw [country flag](https://www.countryflags.io/) for example. Do you have plans to support country code in your csv tables?

**Comment by yetzt** [comment]
I agree. Depending on an ISO standard would also potentially prevent double records.

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
